themes/src/main/resources/theme/keycloak.v2/account/src/app/PageHeaderTool.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+
+import {PageHeaderTools} from '@patternfly/react-core';
+import {ReferrerLink} from './widgets/ReferrerLink';
+import {LogoutButton} from './widgets/Logout';
+
+declare const referrerName: string;
+
+export class PageHeaderTool extends React.Component {
```

Comment:
Class style components are on the way out, can you refactor this to a function component? See the [React documentation](https://reactjs.org/docs/components-and-props.html).